package.json

```diff
@@ -56,33 +56,34 @@
     "contributors:generate": "all-contributors generate"
   },
   "devDependencies": {
-    "@auto-it/exec": "^10.42.2",
-    "@auto-it/first-time-contributor": "^10.42.2",
-    "@auto-it/released": "^10.42.2",
-    "@auto-it/upload-assets": "^10.42.2",
-    "@auto-it/vscode": "^10.42.2",
-    "@types/node": "^18.14.2",
-    "@vscode/test-electron": "^2.2.3",
-    "all-contributors-cli": "^6.24.0",
-    "auto": "^10.42.2",
-    "esbuild": "^0.16.14",
-    "esbuild-plugin-text-replace": "^1.2.0",
-    "eslint": "^8.31.0",
+    "@auto-it/exec": "^11.0.4",
+    "@auto-it/first-time-contributor": "^11.0.4",
+    "@auto-it/released": "^11.0.4",
+    "@auto-it/upload-assets": "^11.0.4",
+    "@types/node": "^20.8.6",
+    "@vscode/test-electron": "^2.3.5",
+    "@vscode/vsce": "^2.21.1",
+    "all-contributors-cli": "^6.26.1",
+    "auto": "^11.0.4",
+    "esbuild": "^0.19.4",
+    "esbuild-plugin-text-replace": "^1.3.0",
+    "eslint": "^8.52.0",
     "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^27.2.0",
-    "jest": "^29.3.1",
-    "jest-environment-node": "^29.3.1",
-    "ovsx": "^0.7.1",
-    "prettier": "^2.8.1",
+    "eslint-plugin-import": "^2.28.1",
+    "eslint-plugin-jest": "^27.4.2",
+    "jest": "^29.7.0",
+    "jest-environment-node": "^29.7.0",
+    "ovsx": "^0.8.3",
+    "prettier": "^3.0.3",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.5",
+    "typescript": "^5.2.2",
```

Comment:
If you update TypeScript then you have to update the examples. Rather we do the least amount of changes